### PR TITLE
[synthetics] Add information about the `push` command's `--yes` option

### LINKED ISSUE #2220: [Synthetics] Document additional Synthetics CLI options
## Description

Some additional capabilities have been added to the Synthetics CLI that need to be documented.  These should be added to the existing [command reference](https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html) page.

The additional features are:
* Delete all monitors from a project
  * Implemented in [synthetics#581](https://github.com/elastic/synthetics/issues/581) (the description in the [PR](https://github.com/elastic/synthetics/pull/598) more accurately shows how this works)
  * When pushing a project that no longer contains any monitors, the UI will prompt the user if they wish to delete all monitors
    * They have to select _yes_ for them to be deleted
  * If running non-interactively, this can be overridden on the CLI with the `--yes` option (implemented in [synthetics#601](https://github.com/elastic/synthetics/issues/601))
* Override pushing a project to a different project id
  * If you have pushed a folder with a project id (e.g. `apples`) and then try and push it to a different id (e.g. `bananas`), then the user will be prompted with the message
    > Monitors were pushed under the 'apples' project. Are you sure you want to push them under the new 'bananas' (note that this will duplicate the monitors, the old ones being orphaned)
  * If running non-interactively, this can be overridden on the CLI with the same `--yes` option mentioned above
* Discover locations from the CLI
  * Although `init` will present you with a list of locations, this is only on the first initialisation of the project folder
  * There is a new `locations` command that can be used at any time, to get a list of locations (implemented in [synthetics#571](https://github.com/elastic/synthetics/pull/571))
  * If just running `npx @elastic/synthetics locations`, a list of public locations **known to that version of the Synthetics Agent** will be shown (i.e. this may be outdated, if additional locations are added, but the Synthetics Agent has not been updated)
  * You can also run with `--auth` and `--url` parameters, to get a full list of the public locations, as well as any private locations, known to Kibana, e.g. `npx @elastic/synthetics locations --auth [auth_token] --url [kibana_url]"`

## Collaboration

* The docs team will lead producing the content with assistance from @vigneshshanmugam or @paulb-elastic 

## Suggested Target Release

* 8.5.0

## Stakeholders

* @vigneshshanmugam 
* @paulb-elastic 
* @andrewvc

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Closes #2220 

## Overview

Updates [Synthetics command reference](https://observability-docs_2246.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-command-reference.html) with changes to the Elastic Synthetics CLI:

* Adds a note letting the reader know that there are interactive prompts, why they are there, and when they should expect to see them. (Note: I'm not sure if this is necessary.)
* Adds a `--yes` option and explains how it works for both deleting all monitors and duplicating monitors to a new ID.

Note: The `locations` command was documented in https://github.com/elastic/observability-docs/pull/2116. Let me know if there are any updates to make to the existing language!

## For the reviewer

I'm very open to suggestions about the language used here!

### CODE REVIEW

**Review by dedemorton — APPROVED** [review]
LGTM

File: docs/en/observability/synthetics-command-reference.asciidoc [hunk 1]
```diff
@@ -81,6 +81,17 @@ Create monitors in {kib} by using your local journeys.
...
 ----
 
+[NOTE]
+====
+The `push` command includes interactive prompts to prevent you from accidentally deleting or duplicating monitors.
+You will see a prompt when:
+
+* You `push` a project that used to contain one or more monitors but
+no longer contains any monitors. Select `yes` to delete them all.
+* You `push` a project that's already been pushed using one ID and then try to `push`
```
Comment (vigneshshanmugam) [inline docs/en/observability/synthetics-command-reference.asciidoc:91]:
```suggestion
* You `push` a project that's already been pushed using one project ID and then try to `push`
```

File: docs/en/observability/synthetics-command-reference.asciidoc [hunk 2]
```diff
@@ -81,6 +81,17 @@ Create monitors in {kib} by using your local journeys.
 npx @elastic/synthetics push --auth <api-key> --url <kibana-url> --project <id|name>
 ----
 
+[NOTE]
+====
+The `push` command includes interactive prompts to prevent you from accidentally deleting or duplicating monitors.
+You will see a prompt when:
+
+* You `push` a project that used to contain one or more monitors but
```
Comment (vigneshshanmugam) [inline docs/en/observability/synthetics-command-reference.asciidoc:89]: Might need to reword to make it clear that `Monitors that are part of a particular project-id gets deleted` not all of them.

File: docs/en/observability/synthetics-command-reference.asciidoc [hunk 3]
```diff
@@ -102,6 +112,16 @@ A unique id associated with your project.
 It will be used for logically grouping monitors.
 If you used <<elastic-synthetics-init-command, `init` to create a project>>, this is the `<name-of-project>` you specified.
 
+`--yes`::
```
Comment (paulb-elastic) [inline docs/en/observability/synthetics-command-reference.asciidoc:115]: Suggest duplicating the changes suggested above ([here](https://github.com/elastic/observability-docs/pull/2246/files#r992685103) and [here](https://github.com/elastic/observability-docs/pull/2246/files#r992684298)) by Vignesh, in the `--yes` section too

**Review by paulb-elastic — APPROVED** [review]
Subject to Vignesh's comments, LGTM

**Pull request merged**